.vscode/settings.json

```diff
@@ -1,16 +1,17 @@
-// If one would like to add/remove/modify user preferences without modifying the content of the
-// workspace settings file, then one would need to modify the `settings.json` under here:
-// - Windows: %APPDATA%\Code\User\settings.json
-// - Linux: $HOME/.config/Code/User/settings.json
-// - Mac: $HOME/Library/Application Support/Code/User/settings.json
 {
-  "tslint.enable": true,
-  "editor.formatOnSave": true,
-  "search.exclude": {
-    "**/node_modules": true,
-    "**/lib": true,
-    "**/coverage": true
+  "[json]": {
```

Comment:
Could you elaborate why so many changes here?

Reply:
I added a preference, auto-formatting applied the "2 spaces tab" for JSON files. It makes it more consistent with our editor settings so I left this change in.

Reply:
whitespaces are fine, i don't get why settings are rearranged?

Reply:
I have no idea actually, will revert and redo the changes there.